chat_server/package.json

```
@@ -31,5 +31,8 @@
     "prettier": "^2.0.5",
     "socket.io": "^2.3.0",
     "socketio-jwt": "^4.5.0"
+  },
+  "prettier": {
+    "tabWidth": 2
```

```suggestion
  }
```
The tabWidth is 2 by default